packages/core/package.json

```diff
@@ -8,12 +8,19 @@
   "module": "dist/browser/asciidoctor.js",
   "exports": {
     "node": {
-      "import": "./dist/node/asciidoctor.js",
-      "require": "./dist/node/asciidoctor.cjs"
+      "import": {
+        "default": "./dist/node/asciidoctor.js",
+        "types": "./types/index.d.ts"
+      },
+      "require": {
+        "default": "./dist/node/asciidoctor.cjs",
+        "types": "./types/index.d.ts"
```

Comment:
It seems that we need to specify a `d.cts` types file (CommonJS). Since we are using `export default function` I don't think we should specify `types` on `require`.

Reply:
You may be right regarding the `d.cts` extension, seeing https://github.com/microsoft/TypeScript/issues/52363
But if you look at the answers in the same thread, I think there's still need to specify the `types` explicitly on the `require` conditional, as there's not matching patch TypeScript will not load it.

My current project is configured for ESM so I could not test the CJS part, it would be helpful to check it on a real CJS project. I'll try to build a test project for that next monday

Reply:
Thanks that would be helpful!
If it works on CJS we can proceed and merge.